app/assets/stylesheets/global.sass

```diff
@@ -2764,3 +2764,32 @@ nav.navbar.navbar-default ul.dropdown-menu.cycle
   font-weight: bold
   text-align: center
   line-height: 20px
+
+.smart-banner
+  color: #fff
+  font-family: "Merriweather"
+  margin-top: 38px
+
+  @media (min-width: $screen-sm-min)
+    margin-top: 0px
+
+  .info
+    padding: 10px
+    background: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7))
```

Comment:
You are not using a gradient here, just set the background color to `rgba(0, 0, 0, 0.7)`.